docs/source/index.rst

```diff
@@ -4,6 +4,15 @@ Hyperledger Iroha documentation
 
 .. image:: ../image_assets/iroha_logo.png
 
+.. warning::
+    Please note that Hyperledger Iroha v1 is **no longer under active development** and the support for Iroha v1 is limited. The core development team focuses on `Hyperledger Iroha v2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_, a complete rewrite of Iroha in Rust.
```

Comment:
```suggestion
    Please note that Hyperledger Iroha v1 is **no longer under active development** therefore, support for Iroha v1 is limited. The core development team is focused on `Hyperledger Iroha v2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_, a complete rewrite of Iroha in Rust.
```

Reply:
shouldn't there be a semicolon before "therefore"?